packages/web-components/src/components/cbp-banner/cbp-banner.stories.tsx

```diff
@@ -0,0 +1,44 @@
+export default {
+    title: 'Components/Banner',
+    tags: ['autodocs'],
+    argTypes: {
+        bannerTitle: {
+          name: 'Banner title',
+          description: 'Text to fill the banner h2',
+          control: 'text',
+        },
+        bannerContent:{
+          name: 'Banner Content',
+          description: 'Text to fill the banner Content',
+          control: 'text',
+        },
+    },
+      args: {
+      },
+    };
+  
+  const Template = ({ bannerTitle, bannerContent}) => {
+    return ` 
+        <cbp-banner 
+            bannerTitle= ${bannerTitle}
+            bannerContent= ${bannerContent}
+        >
+            <cbp-typography 
+                slot='cbp-banner-title'
+                tag='h2'
```

Comment:
I'm not sure that h2 makes sense, since this is displayed before the h1. Let's just make it a div. The variant is giving it the visual styling.